packages/cli-doctor/src/tools/healthchecks/adb.ts

```diff
@@ -0,0 +1,52 @@
+import {HealthCheckInterface} from '../../types';
+import {
+  adb,
+  getAdbPath,
+  tryRunAdbReverse,
+  listAndroidDevices,
+} from '@react-native-community/cli-platform-android';
+import {execFileSync} from 'child_process';
+
+export default {
+  label: 'Adb Devices',
+  description: 'Required to verify if the android device is attached correctly',
+  getDiagnostics: async () => {
+    const adbPath = getAdbPath();
+    const devices = adb.getDevices(adbPath);
+
+    if (devices.length > 0) {
+      const adbArgs = ['reverse', '--list'];
+      const reverseList = execFileSync(adbPath, adbArgs, {encoding: 'utf8'});
+      if (reverseList.length > 0) {
+        return {
+          needsToBeFixed: false,
+        };
+      } else {
+        return {
+          description:
+            'The reverse proxy for the Android device has not been set.',
+          needsToBeFixed: true,
+        };
+      }
+    } else {
+      return {
+        description:
+          'No devices and/or emulators connected. Please create emulator with Android Studio or connect Android device.',
+        needsToBeFixed: true,
+      };
+    }
+  },
+  runAutomaticFix: async ({loader, logManualInstallation}) => {
+    loader.fail();
+    const device = await listAndroidDevices();
+    if (device && device.connected) {
+      tryRunAdbReverse(8081, device.deviceId);
```

Comment:
I'm not sure is it good way, but maybe let's use something like this here?
```suggestion
      tryRunAdbReverse(process.env.RCT_METRO_PORT || 8081, device.deviceId);
```